include/renderer.hpp

```diff
@@ -21,6 +21,20 @@ enum class RendererType : s8 {
 };
 
 struct EmulatorConfig;
+enum class ShaderMode : s8 {
```

Comment:
(It's probably better to not have a type specification here and let the compiler pick, not like this is really getting used as a numerical value)